### PR TITLE
Improve Docs In `azure_rm_virtualnetwork`

### PR DESCRIPTION
**Pull request by timway**
##### SUMMARY
* The DNS Servers should be a list of strings
* Remove the trailing comma in the `dict()` call for `dns_servers`
* The boolean values should default to a boolean value in the docs
* Clean up excess quotation in some of the examples

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
```bash
azure_rm_virtualnetwork
```

##### ADDITIONAL INFORMATION

```bash
ansible-doc --type module azure.azcollection.azure_rm_virtualnetwork
```

### CODE REVIEW

File: plugins/modules/azure_rm_virtualnetwork.py [hunk 1]
```diff
@@ -218,7 +222,7 @@ def __init__(self):
             state=dict(type='str', default='present', choices=['present', 'absent']),
             location=dict(type='str'),
             address_prefixes_cidr=dict(type='list', aliases=['address_prefixes']),
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_virtualnetwork.py]:
```suggestion
            address_prefixes_cidr=dict(type='list', elements='str', aliases=['address_prefixes']),
```

File: plugins/modules/azure_rm_virtualnetwork.py [hunk 2]
```diff
@@ -218,7 +222,7 @@ def __init__(self):
...
-            dns_servers=dict(type='list',),
+            dns_servers=dict(type='list'),
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_virtualnetwork.py]:
```suggestion
            dns_servers=dict(type='list', elements='str'),
```

File: plugins/modules/azure_rm_virtualnetwork.py (hunk 1 above)
Comment (timway) [reply plugins/modules/azure_rm_virtualnetwork.py]: @Fred-sun I've updated the code to reflect this change. Thank you for the comment!

File: plugins/modules/azure_rm_virtualnetwork.py (hunk 2 above)
Comment (timway) [reply plugins/modules/azure_rm_virtualnetwork.py]: @Fred-sun I've updated the code to reflect this change. Thank you for the comment!

**Comment by Fred-sun** [comment]
@timway Please delete the corresponding line in the following file
```e.g.
Line 228 -- test/sanity/ignore-2.10.txt  plugins/modules/azure_rm_virtualnetwork.py validate-modules:parameter-list-no-elements
Line 228 -- test/sanity/ignore-2.11.txt  plugins/modules/azure_rm_virtualnetwork.py validate-modules:parameter-list-no-elements
Line 228 -- test/sanity/ignore-2.12.txt  plugins/modules/azure_rm_virtualnetwork.py validate-modules:parameter-list-no-elements
Line 227 -- test/sanity/ignore-2.13.txt  plugins/modules/azure_rm_virtualnetwork.py validate-modules:parameter-list-no-elements
Line 224 -- test/sanity/ignore-2.14.txt  plugins/modules/azure_rm_virtualnetwork.py validate-modules:parameter-list-no-elements

**Comment by Fred-sun** [comment]
@timway Please update the corresponding Ignore*.txt, because we added elements in document and args, which should be deleted （ignore line)accordingly! Thank you very much!

**Comment by Fred-sun** [comment]
kindly ping!

**Comment by timway** [comment]
@Fred-sun I believe I addressed the ignores in the most recent commit.

**Comment by Fred-sun** [comment]
@timway Sorry, it has been added, I will push forward the merge.

**Comment by Fred-sun** [comment]
@xuzhang3  ready_for_review

**Review by xuzhang3 — APPROVED** [review]

**Pull request merged**